pkg/cmd/install.go

```diff
@@ -269,12 +272,8 @@ func (o *installCmdOptions) install(cobraCmd *cobra.Command, _ []string) error {
 
 		kanikoBuildCacheFlag := cobraCmd.Flags().Lookup("kaniko-build-cache")
 
-		defaultKanikoBuildCache := true
-
 		if kanikoBuildCacheFlag.Changed {
 			platform.Spec.Build.KanikoBuildCache = &o.kanikoBuildCache
-		} else {
-			platform.Spec.Build.KanikoBuildCache = &defaultKanikoBuildCache
```

Comment:
Can you confirm it is intended to be part of that PR?

Reply:
Yes, I've removed setting the default value in spec, because it's computed later in the platform reconcile cycle and its value is now set into the status.

Reply:
Ah right!